### PR TITLE
New dress and service uniform!

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

Resprite dress uniform to what #267 intended to be (HOLY SMOKES, A PR THAT OLD), but #3682 deleted them and gave us cringe dress uniform that make marines look bad.

Also, adding in service uniform! Sprites by Fitz 'Pancake' Sholl!

ALSO, making it easier to vent out dress and service uniform.

## Why It's Good For The Game

Dress and service uniform make a marine prideful of serving Uncle Sam, or for TGMC lore SPACE SAM. If it looks ugly, why even wear it?

## Changelog
:cl: nova_epee and Fitz 'Pancake' Sholl
fix: Dress uniform to the intended sprites. I choose not to delete the old dress uniform.
imageadd: Add service uniform; say thanks to Fitz 'Pancake' Sholl!
add: Make it easy to get service and dress uniform without having to click so much buttons.
/:cl:

### CODE REVIEW

**Comment by novaepee** [comment]
Sprites before and after

image

**Comment by novaepee** [comment]
Here is the service uniform! Wear those boots.

image

**Comment by AndroidMK2** [comment]
I like the new service uniform but the white pants on the dress uniform don't fit as much as the black pants from before.

**Comment by Verlurk** [comment]
New dress uniform looks like a shapeless blob because none of it is broken up. Former looks better, in my opinion.

Service looks nice though. Do these actually have a purpose for a role, or is it just excess vendor?

**Comment by Tennessee116** [comment]
If you're going to use the ICCG dress uniform, at least port the boots.

**Comment by novaepee** [comment]
> I like the new service uniform but the white pants on the dress uniform don't fit as much as the black pants from before.

They are baggy, which is an interesting choice from the spriters.

**Comment by novaepee** [comment]
> Do these actually have a purpose for a role, or is it just excess vendor?

Excess vendor.

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 1]
```diff
@@ -1091,10 +1091,8 @@
 			/obj/item/clothing/head/tgmcberet = -1,
 			/obj/item/clothing/glasses/mgoggles = -1,
 			/obj/item/clothing/glasses/mgoggles/prescription = -1,
-			/obj/item/clothing/under/whites = -1,
-			/obj/item/clothing/head/white_dress = -1,
-			/obj/item/clothing/shoes/white = -1,
-			/obj/item/clothing/gloves/white = -1,
+			/obj/effect/essentials_set/white_dress = -1,
```
Comment (BraveMole) [inline code/game/objects/machinery/vending/marine_vending.dm:1094]: no. This makes it not savable in loadouts

**Review by BraveMole — CHANGES_REQUESTED** [review]

**Comment by BraveMole** [comment]
Stale. Get kuro to open and review this

**Review by Kuro020 — APPROVED** [review]
I JUST WANT TO PLAY TERRARIA FOR GODS SAKE

**Comment by Luxyboo** [comment]
> I JUST WANT TO PLAY TERRARIA FOR GODS SAKE

lmao

**Comment by FosterGrime-Heymynameslouie** [comment]
can we get the old dress uniform back in the form of a seperate uniform?

**Pull request merged**